### PR TITLE
fix ticker without stop

### PR DESCRIPTION
**Pull request by rokkiter**
**Please provide a description of this PR:**

fix create time.NewTicker but not stop

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

**Review by hanxiaop — APPROVED** [review]

File: pilot/pkg/model/jwks_resolver_test.go [hunk 1]
```diff
@@ -292,6 +292,7 @@ func TestJwtPubKeyEvictionForNotRefreshed(t *testing.T) {
 		for {
 			select {
 			case <-done:
+				c.Stop()
```
Comment (hzxuzhonghu) [inline pilot/pkg/model/jwks_resolver_test.go:295]: nit: why not defer, so we will never forget `Stop` it

Comment (rokkiter) [reply pilot/pkg/model/jwks_resolver_test.go:295]: You are right, it is more reasonable to use defer here

**Pull request merged**